README.md

```diff
@@ -99,54 +101,67 @@ This will start the app on port `8080` and make it accessible at [http://localho
 The docker-compose setup starts two containers: 
 
 * `threats-client`: running `nginx` as a reverse proxy and serving the react application
-* `threats-server`: running the `nodejs` backends: public API and game server
+* `threats-server`: running the Node.js backend: public API and game server
 
 ![docker-compose setup](docs/docker-setup.svg)
 
 #### Local deployment
 
-The server can be started using:
+To start both the server and UI in dev mode, just run
 
 ```bash
-npm run server
+npx turbo run dev
```

Comment:
Unfortunately, this does not work for me... 

I get an error
```
 WARNING  No locally installed `turbo` found. Using version: 2.0.6.
• Packages in scope: @eop/client, @eop/cornucopia-cards, @eop/eslint-config, @eop/prettier-config, @eop/server, @eop/shared, @eop/typescript-config
• Running dev in 7 packages
• Remote caching disabled
@eop/shared:build: cache miss, executing e8910eb161c82f2e
@eop/shared:build: 
@eop/shared:build: > @eop/shared@0.24.0 build
@eop/shared:build: > rimraf dist *.tsbuildinfo && tsc --build tsconfig.lib.json
@eop/shared:build: 
@eop/shared:build: tsconfig.lib.json:3:14 - error TS6053: File '@eop/typescript-config/base.json' not found.
@eop/shared:build: 
@eop/shared:build: 3   "extends": "@eop/typescript-config/base.json",
@eop/shared:build:                ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
@eop/shared:build: 
@eop/shared:build: 
@eop/shared:build: Found 1 error.
@eop/shared:build: 
@eop/shared:build: npm ERR! Lifecycle script `build` failed with error: 
@eop/shared:build: npm ERR! Error: command failed 
@eop/shared:build: npm ERR!   in workspace: @eop/shared@0.24.0 
@eop/shared:build: npm ERR!   at location: /[...]/elevation-of-privilege/packages/shared 
@eop/shared:build: 
@eop/shared:build: ERROR: command finished with error: command (/[...]/elevation-of-privilege/packages/shared) /usr/bin/npm run build exited (1)
@eop/shared#build: command (/[...]/elevation-of-privilege/packages/shared) /usr/bin/npm run build exited (1)

 Tasks:    0 successful, 1 total
Cached:    0 cached, 1 total
  Time:    3.04s 
Failed:    @eop/shared#build

 ERROR  run failed: command  exited (1)
```

with the versions
```
►node --version
v20.11.1
►npm --version
10.2.4
►npx --version 
10.2.4
```

Reply:
Did you run npm install in the root directory first?

Reply:
Of course, not. #facepalm

sorry for that noise....  Now it's working